components/PeerReviews/PeerReviewInviteModal.tsx

```diff
@@ -0,0 +1,139 @@
+import { ReactElement, useRef, useState } from "react";
+import { css, StyleSheet } from "aphrodite";
+import { PeerReviewRequest } from "~/config/types/peerReview";
+import BaseModal from "~/components/Modals/BaseModal";
+import FormInput from "~/components/Form/FormInput";
+import Button from "~/components/Form/Button";
+import { inviteReviewer } from "./config/PeerReviewInviteAPI";
+import { MessageActions } from "~/redux/message";
+import { connect } from "react-redux";
+import Loader from "~/components/Loader/Loader";
+import colors from "~/config/themes/colors";
+
+type Props = {
+  peerReviewRequest: PeerReviewRequest;
+  fetchReviewRequests: Function;
+  isOpen: Boolean;
+  closeModal: Function;
+  setMessage: Function;
+  showMessage: Function;
```

Comment:
It's pretty nice if you're able to alphabetize these. Short cut on VS code is highlight-lines then fn+f9